src/views/Welcome/App.tsx

```diff
@@ -289,100 +304,155 @@ export default function App() {
     };
   }
 
+  const SetupTemplate = ({
```

Comment:
If you have components that are more than 4-5 lines and you reuse them, please create a different file for it in the components directory